packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/RiskInvestment/Lines.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Button as ConstellationButton } from '@blockchain-com/constellation'
+
+import { Text } from 'blockchain-info-components'
+
+import { ListContainer } from './Lines.styles'
+
+type Props = {
+  lines: string[]
+}
+
+export const Lines = ({ lines }: Props) => {
+  const [isCollapsed, setIsCollapsed] = useState(true)
+
+  const toggleCollapse = () => {
+    setIsCollapsed(!isCollapsed)
+  }
+
+  return (
+    <div>
+      <ListContainer isCollapsed={isCollapsed}>
```

Comment:
I'm not sure if it was in v4 or v5, but linter didn't like props with camelCase. If it's the case, prepend a `$` it to make it a [transient prop](https://styled-components.com/docs/api#transient-props)